moviefind/src/js/components/Movie.js

```diff
@@ -1,28 +1,30 @@
-var React =require('react');
+var React = require('react');
 var AppActions = require('../actions/AppActions');
 var AppStore = require('../stores/AppStore');
 
 var Movie = React.createClass({
 
-	render: function(){
+	render: function () {
 		var link = 'https://www.imdb.com/title/' + this.props.movie.imdbID;
-		return( 
-			<div className ="well">
-				<div className ="row">
-					<div className ="col-md-4">
-					<img className="thumbnail" src = {this.props.movie.Poster}/>
+		return (
+			<div className="well">
+				<div className="row">
+					<div className="col-md-4">
+						<img className="thumbnail" src={this.props.movie.Poster} />
 					</div>
 
```

Comment:
This PR introduces a potential bug. In the `Movie` component, you're trying to access `this.props.movie.name` and `this.props.movie.info` which don't exist in the `movie` object. This will likely lead to `undefined` being rendered. Make sure these properties are actually available in the `movie` object or handle cases where they might be missing.